### PR TITLE
fix:replace Comment hl to SagaVirtLine

### PR DESCRIPTION
**Pull request by Mr-LLLLL**

### CODE REVIEW

File: lua/lspsaga/diagnostic/init.lua [hunk 1]
```diff
@@ -278,10 +278,10 @@ function diag:render_diagnostic_window(entry, option)
 
   local virt = {}
   if entry.source then
-    virt[#virt + 1] = { entry.source, 'Comment' }
+    virt[#virt + 1] = { entry.source, 'SagaVirtLine' }
```
Comment (glepnir) [inline lua/lspsaga/diagnostic/init.lua:281]: this is not virtual line.  so use `SagaVirtLine` a bit inappropriate

Comment (Mr-LLLLL) [reply lua/lspsaga/diagnostic/init.lua]: thank you for review

File: lua/lspsaga/finder/box.lua [hunk 2]
```diff
@@ -173,7 +173,7 @@ function M.indent(ns, lbufnr, lwinid)
           virt_text_pos = 'overlay',
           ephemeral = true,
         })
-        api.nvim_set_hl(0, hi, { link = 'Comment', default = true })
+        api.nvim_set_hl(0, hi, { link = 'SagaVirtLine', default = true })
```
Comment (glepnir) [inline lua/lspsaga/finder/box.lua:176]: there is for finder indent . so maybe it's better not changed here or can we give it `SagaIndent` ?

File: lua/lspsaga/symbol/outline.lua [hunk 3]
```diff
@@ -133,7 +133,7 @@ function ot:parse(symbols, bufnr, list)
 
       if config.outline.detail then
         buf_set_extmark(bufnr, ns, row - 1, 0, {
-          virt_text = { { node.detail or '', 'Comment' } },
+          virt_text = { { node.detail or '', 'SagaVirtLine' } },
```
Comment (glepnir) [inline lua/lspsaga/symbol/outline.lua:136]: there is better use `SagaDetail` create it in highlight.lua and link to `Comment`

File: lua/lspsaga/implement/init.lua [hunk 4]
```diff
@@ -77,7 +77,7 @@ local function try_render(client_id, bufnr, pos, data)
 
     api.nvim_buf_set_extmark(bufnr, ns, pos.line, 0, {
       id = data.virt_id,
-      virt_lines = { { { indent .. #result .. ' ' .. word, 'Comment' } } },
+      virt_lines = { { { indent .. #result .. ' ' .. word, 'SagaSpinner' } } },
```
Comment (glepnir) [inline lua/lspsaga/implement/init.lua:80]: there no need change.

**Pull request closed without merging**